### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by sctnightcore**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -344,84 +337,3 @@ proprietary programs.  If your program is a subroutine library, you may
 consider it more useful to permit linking proprietary applications with the
 library.  If this is what you want to do, use the GNU Lesser General
 Public License instead of this License.
-
```
Comment (lututui) [inline LICENSE]: Bring back this part

Comment (sctnightcore) [reply LICENSE]: Delete LICENSE.md : a3ba25b

Comment (sctnightcore) [reply LICENSE]: Wait :p

Comment (sctnightcore) [reply LICENSE]: Fixed : eb9d103

**Pull request merged**